cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -258,7 +258,8 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
                          @QueryParam("pageToken") String pageToken,
                          @QueryParam("pageSize") Integer pageSize,
                          @QueryParam("orderBy") SortOrder orderBy,
-                         @QueryParam("nameFilter") String nameFilter
+                         @QueryParam("nameFilter") String nameFilter,
+                         @QueryParam("latestOnly") String latestOnly
```

Comment:
not sure if queryParam can get boolean directly or it reads it as string?

Reply:
latest can be a string